notion/blockupdater.go

```diff
@@ -51,8 +51,10 @@ func (b *blockWithChild) Children() *notionapi.Blocks {
 		return &bl.BulletedListItem.Children
 	case *notionapi.NumberedListItemBlock:
 		return &bl.NumberedListItem.Children
+	case *notionapi.CodeBlock:
+		return nil
 	default:
-		panic(fmt.Sprintf("unknown block type: %T", b))
+		panic(fmt.Sprintf("unknown block type: %T", b.b))
```

Comment:
This one caused me to facepalm for a few seconds 😆

Reply:
I suggest creating a helper in this package (or maybe exporting the other one) for constructing a unified "unknown block" panic; in another PR I added:

```go
	panic(errUnknownBlock(block, nil))
```

```go
func errUnknownBlock(block notionapi.Block, data any) error {
	raw, _ := json.Marshal(data)
	return fmt.Errorf("unknown block type: %T (%s)", block, string(raw))
}
```

Makes it easy to always have lots of details for diagnostics